docs/guides/gke_github.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Docs are generated from their templates. Have a look at the Github guid for an example of how this is done. The templates allows you to write text and refer to files in the examples repo.
https://github.com/fluxcd/terraform-provider-flux/blob/main/templates/guides/github.md.tmpl

Reply:
Added! https://github.com/fluxcd/terraform-provider-flux/pull/112/commits/f2aecbd113dab1b3fe2f428cbc8698f21f97de03